core/src/services/alluxio/core.rs

```diff
@@ -197,16 +213,27 @@ impl AlluxioCore {
 
         match status {
             StatusCode::OK => Ok(()),
-            _ => Err(parse_error(resp).await?),
+            _ => {
+                let err = parse_error(resp).await?;
```

Comment:
How about check status code directly so that we don't need to parse the response while meeting 404?

Reply:
when alluxio  resource  not found, it return 500 status code

Reply:
Got it.